### DISCUSSION TITLE
When will supervised contact prediction model be released?

### DISCUSSION THREAD

**Discussion by strategist333** [facebookresearch/esm#59] [General]
Am I correct to interpret the "Coming Soon" for ESM-MSA-1 in the supervised contact prediction table on the README as an indication of plans to release the pretrained supervised contact prediction model? If so, when will that be released?

**Comment by tomsercu** [comment]
The "coming soon" just means that we are working on providing a more complete table with updated supervised contact prediction benchmarking of all those models.
We don't have a concrete timeline for releasing the supervision pipelines.
